backend/python/app/models/entity_mg.py

```diff
@@ -0,0 +1,18 @@
+from mongoengine import *
+from .enum import enum
+
+
+class Entity(Document):
+    string_field = StringField(max_length=200, required=True)
```

Comment:
Any reason for capping it at 200 characters? If not, let's leave it unlimited, project teams can adjust based on their needs 🙂